.github/ISSUE_TEMPLATE/bug_report_issue_template.yml

```diff
@@ -50,8 +47,8 @@ body:
   - type: textarea
     id: logs
     attributes:
-        Please copy and paste any relevant log output. This will be
-        automatically formatted into code, so no need for backticks.
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
```

Comment:
Is it expected that the bug report template is changed with this PR?

Reply:
I think I'll just look into manually changing that back